DirectProgramming/DPC++FPGA/Tutorials/Features/ac_types/ap_float/README.md

```diff
@@ -0,0 +1,471 @@
+| Hardware                          | Intel® Programmable Acceleration Card (PAC) with Intel Arria® 10 GX FPGA <br> Intel® FPGA Programmable Acceleration Card (PAC) D5005 (with Intel Stratix® 10 SX) <br> Intel® FPGA 3rd party / custom platforms with oneAPI support <br> *__Note__: Intel® FPGA PAC hardware is only compatible with Ubuntu 18.04* 
+| Software                          | Intel® oneAPI DPC++ Compiler <br> Intel® FPGA Add-On for oneAPI Base Toolkit
+| What you will learn               | Including and using the `ap_float` type <br> Using `ap_float` type to trade off mathematical accuracy for lesser resource utilization <br> Using various `ap_float` rounding modes and their effect on accuracy and resource utilization <br> Using the `ap_float `math functions for better quality of results
+| Time to complete                  | 1 hour
+
+## Purpose
+
+This FPGA tutorial shows how to use the `ap_float` type with some simple examples and recommended best practices.
+
+This data-type can be used in place of native floating point types to generate area efficient and optimized designs for the FPGA. For example, operations which do not utilize all of the bits the native types or desings which do not require all of the range and precision of native types are good candidates for replacement with the `ap_float` type.
+
+This tutorial will present the following:
+1. How to include the `ap_float` type and an overview of common ap_float use cases.
+2. A Polynomial Sine Approximation example which illustrates how to trade off mathematical accuracy for lesser FPGA resource utilization.
+3. Rounding Mode and native type to ap_float type conversion examples which describe various `ap_float` rounding modes and their effect on accuracy and FPGA resource utilization.
+4. A Quadratic Equation Solver example which show cases explicit `ap_float` math functions and how they can be used to replace mathematical operators like `*, /, +` and `-` for better quality of results.
+
+### Simple Code Example
+
+An `ap_float` number can be defined as follows:
+
+```cpp
+ihc::ap_float<EW, MW> a;
+```
+Here EW specifies the exponent width and MW specifies the mantissa width of the number. Optionally, another template parameter can be specified to set the rounding mode. For more details please refer to the section titled `Variable-Precision Integer and Floating-Point Support` in the Intel® oneAPI DPC++ FPGA Optimization Guide.
+
+To use this type in your code, you must include the following header:
+
+```cpp
+#include <sycl/ext/intel/ac_types/ap_float.hpp>
+```
+
+To use `ap_float` math functions, you must include the following header:
+
+```cpp
+#include <sycl/ext/intel/ac_types/ap_float_math.hpp>
+```
+
+Additionally, you must use the flag `-qactypes` in order to ensure that the headers are correctly included and that the compiler links against the necessary libraries for emulation support.
```

Comment:
please tell the user where the `-qactypes` or `/Qactypes` flag appears.

Reply:
Added a note, however I might have the -fintelfpga flag absorb -qactypes before this tutorial is released.